recipes/assembly_stats/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "assembly_stats" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "4393e3007a6d8674ff32348ae33a5e013a38db5452e86ab4b4b86883aab63e96"
```

Could you please remove the quotes from the ``sha256``?
```suggestion
  sha256: 4393e3007a6d8674ff32348ae33a5e013a38db5452e86ab4b4b86883aab63e96
```